src/tools/run-make-support/src/command.rs

```diff
@@ -76,16 +100,47 @@ impl CompletedProcess {
         self.output.status
     }
 
+    /// Checks that trimmed `stdout` matches trimmed `content`.
+    #[track_caller]
+    pub fn assert_stdout_equals(self, content: &str) -> Self {
```

Comment:
Suggestion: should this and other `assert_*` helpers take a `S: AsRef<str>`? That way the call-site could pass an owned `String` if the call-site has a `String` lying around without having to `&String`.

Reply:
Good point, I was thinking the same. I'll modify it.